common/browserapp.cpp

```diff
@@ -28,20 +29,12 @@ bool BrowserApp::captivePortal()
 
 QString BrowserApp::profileName()
 {
-    static QString profileName = QStringLiteral("mozembed");
-    static bool argsChecked = false;
-
-    if (!argsChecked) {
-        const QStringList &arguments = QCoreApplication::arguments();
-
-        if (arguments.contains(QStringLiteral("-profile"))) {
-            int index = arguments.indexOf(QStringLiteral("-profile"));
-            if (index + 1 < arguments.size()) {
-                profileName = arguments.at(index + 1);
-            }
-        } else if (BrowserApp::captivePortal()) {
-            profileName = QStringLiteral("captiveportal");
+    const QStringList &arguments = QCoreApplication::arguments();
+    if (arguments.contains(QStringLiteral("-profile"))) {
+        int index = arguments.indexOf(QStringLiteral("-profile"));
```

Comment:
The same QStringLiteral defined twice.

Reply:
That's bad! Even though was there already, I'll clean it up.